app/templates/account/profile/_profile-info.html

```diff
@@ -1,22 +1,34 @@
+{% from "components/follow_button.html" import follow_button with context %}
+
 <div class="row profile-info">
     <div class="profile-image col-3">
         <img src="static/assets/default-profile-picture.png" class="rounded-circle mx-auto d-block" alt="">
     </div>
     <div class="col-9">
         <div class="profile-user-settings ">
-            <h2 class="profile-user-name">{{ user.username }}</h2>
+            <h2 class="profile-user-name">{{ user.username }}
+                {% if not is_me %}
+                    {% if session.get("logged_in").get("username") in user.followers|map(attribute="username") %}
+                        {{ follow_button(true, user) }}
+                    {% else %}
+                        {{ follow_button(false, user) }}
+                    {% endif %}
+                {% endif %}
+            </h2>
```

Comment:
Same thing here, just put the conditional directly as the parameter

```suggestion
            <h2 class="profile-user-name">{{ user.username }}
                {% if not is_me %}
                    {{ follow_button(session.get("logged_in").get("username") in user.followers|map(attribute="username"), user) }}
                {% endif %}
            </h2>
```